src/main/kotlin/com/github/avrokotlin/avro4k/annotations.kt

```diff
@@ -25,10 +25,6 @@ annotation class AvroName(val value: String)
 @Target(AnnotationTarget.PROPERTY, AnnotationTarget.CLASS)
 annotation class ScalePrecision(val scale: Int, val precision: Int)
 
-@SerialInfo
-@Target(AnnotationTarget.CLASS)
-annotation class AvroInline
```

Comment:
Now it's value classes

Reply:
As this is a breaking change, I would add a deprecation tag at least with one release before removing it.